neither this nor linux/raw.h exists, but seem to not be needed
diff --git a/debugfs.ocfs2/include/main.h b/debugfs.ocfs2/include/main.h
index 89a55ca..dc726eb 100644
--- a/debugfs.ocfs2/include/main.h
+++ b/debugfs.ocfs2/include/main.h
@@ -45,7 +45,6 @@
 #include <arpa/inet.h>
 #include <netinet/in.h>
 #include <signal.h>
-#include <sys/raw.h>
 #include <linux/kdev_t.h>
 #include <inttypes.h>
 #include <utime.h>
diff --git a/fswreck/include/main.h b/fswreck/include/main.h
index 03e1b3d..8260e75 100644
--- a/fswreck/include/main.h
+++ b/fswreck/include/main.h
@@ -43,7 +43,6 @@
 #include <arpa/inet.h>
 #include <netinet/in.h>
 #include <signal.h>
-#include <sys/raw.h>
 #include <linux/kdev_t.h>
 #include <inttypes.h>
 
diff --git a/o2info/operations.c b/o2info/operations.c
index 9c0ba30..b52b54e 100644
--- a/o2info/operations.c
+++ b/o2info/operations.c
@@ -22,7 +22,6 @@
 #define _GNU_SOURCE /* Because libc really doesn't want us using O_DIRECT? */
 
 #include <errno.h>
-#include <sys/raw.h>
 #include <inttypes.h>
 #include <sys/ioctl.h>
 
